x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -289,6 +290,18 @@ export function trainedModelsApiProvider(httpService: HttpService) {
         version: '1',
       });
     },
+    createInferenceEndpoint(
```

Comment:
I think it's worth creating a dedicated API service for inference

Reply:
Fix added in https://github.com/elastic/kibana/commit/c6138c66cd8ac2ec7daf3283a8962e6d49c15911